### ISSUE TITLE
Add Import Date as a viewable field in List and Detail view

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#1298]
It would be nice to have "Import Date" as a field that could be viewed in the List and Detail views. It was added to migrated data and so seems to be available as a field that can be selected from List Settings, but there is not a value in it for newly imported data.

**Comment by RDmitchell** [comment]
@nllong -- so is this date only going to appear for newly imported records, not records that are already imported?

**Comment by nllong** [comment]
existing records will get a date, but the date will be when migration is
run.

**Comment by RDmitchell** [comment]
instance: dev1
SHA: 94df60a
Org: LBNL 28

@nllong -- so I guess the fields are called "Updated (Property)" and "Updated (Tax Lot)"?

I imported covered-building-sample.csv as a spreadsheet, mapped fields to both Tax Lot and Property, here are the resulting "updated" field dates
![image](https://user-images.githubusercontent.com/6314950/33909247-d7c1f216-df3f-11e7-8a12-6424a221076d.png)

I then imported the file "portfolio-manager-sample.csv as a Portfolio manager file, all fields mapped to the Property table.
![image](https://user-images.githubusercontent.com/6314950/33909388-5d1eb6ce-df40-11e7-9469-eaccccf59af0.png)

In the Detail view, the program is showing the created and updated dates for the master record but not the individual files that were imported. This seems ok, but would it also make sense to put those dates in the "Created" field for each file that was imported? ie, the Updated date in the master record in this case would be the Created date for the Portfolio Manager record that was imported.
![image](https://user-images.githubusercontent.com/6314950/33909760-87e646fa-df41-11e7-9bfb-9c0e68123d55.png)

**Comment by RDmitchell** [comment]
@nllong -- I'm not sure what you mean by migration -- are you saying that the migration will figure out when all the different children were imported, and then put those dates into these date fields? Or will the migration just put the date of the migration? If it's the second option, I'm not sure adding a date that a migration done is very useful for existing data. It actually might be confusing. Maybe we should leave it blank for existing data if we can't actually determine when the records were created, depending on what the migration is actually doing.

**Comment by RDmitchell** [comment]
I think one of the original reasons I wanted an "import" date (which is now an updated or created date) was to keep track of when data was updated over time, which means putting that information in the individual file import columns, as well as the master record column. 

don't know how hard that is to do, but it seems like it would be very useful to the user to see that information over time for the data they have imported.

**Comment by nllong** [comment]
The created/updated date are attached to the property views... so it won't
show up in the history of the imports. The created is the first time that
the property was created (not the propertystate) and updated was the last
time the property was changed.

When we deploy this on production, all of the updated/created times will be
the date/time of when the migration runs.

**Comment by RDmitchell** [comment]
but what does the migration do? If it doesn't have anything to do with when the records were actually created or updated, I don't think we should add it? It wouldn't make sense to anyone looking at their data. 

Also, I would reiterate that I think those dates should also be associated with the history of the imported records. I wasn't very explicit in my description, but it would probably have been useful to have a design discussion before starting the implication. 

I guess I'll add this request to another issue that we may get to at some point.

**Comment by nllong** [comment]
The migration adds the created and updated fields to the database and sets
the default.

Yeah, having the created/updated on the imported records would be nice, but
I don't see much value other than to see when a file was imported. The goal
of the created/updated was to see when the property was initially created
(will be right for all new records) and when the property was edited.

**Comment by RDmitchell** [comment]
closing
